webapp/src/app/shared/services/filter-management.service.ts

```diff
@@ -289,7 +289,7 @@
        }
    
        let filterTagsData = await this.getFilterTagsData(payload, currentFilterType.optionURL);
-       filterTagsData = updateFilterTags(filterTagsData, currentFilterType.optionName);
+       if(updateFilterTags) filterTagsData = updateFilterTags(filterTagsData, currentFilterType.optionName);
```

Comment:
not sure about TS code style standards but let's have condition and its code on different lines

Reply:
Done.